nodecore-spv/src/main/java/veriblock/service/impl/Blockchain.java

```diff
@@ -95,7 +95,11 @@ public void addAll(List<VeriBlockBlock> blocks) throws SQLException {
 
         List<StoredVeriBlockBlock> storedBlocks = convertToStoreVeriBlocks(listToStore);
 
-        blockStore.put(storedBlocks);
+        try {
+            blockStore.put(storedBlocks);
+        } catch (Exception ex){
+            System.out.printf("" + ex.getMessage(), ex);
```

Comment:
```suggestion
            logger.error("Error while storing blocks", ex);
```
And maybe rethrow as an illegal state exception?

Reply:
Aah... I added this catch for debugging.